src/mono/netcore/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Task/Sdk/Sdk.props

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+
+  <PropertyGroup>
+    <_AOTTasksDirectoryRoot Condition="'$(_AOTTasksDirectoryRoot)' == ''">$(MSBuildThisFileDirectory)../tools/</_AOTTasksDirectoryRoot>
```

Comment:
Why is this needed? This is part of the nuget, and should always be using the bundled task.

Reply:
Removing the file as the NuPkg will no longer import the UsingTask.